nilearn/datasets/atlas.py

```diff
@@ -351,26 +440,31 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
     opts = {'uncompress': True}
     root = os.path.join('data', 'atlases')
 
-    if atlas_name[0] == 'c':
-        if 'cort-maxprob' in atlas_name and symmetric_split:
-            split_name = atlas_name.split('cort')
-            atlas_name = 'cortl' + split_name[1]
-            label_file = 'HarvardOxford-Cortical-Lateralized.xml'
-            lateralized = True
-        else:
-            label_file = 'HarvardOxford-Cortical.xml'
+    if source_idx == 0:
+        if atlas_name[0] == 'c':
             lateralized = False
+            if 'cort-maxprob' in atlas_name or 'cortl-maxprob' in atlas_name and symmetric_split:
+                split_name = atlas_name.split('cort')
+                atlas_name = 'cortl{}'.format(split_name[1])
+                label_file = 'HarvardOxford-Cortical-Lateralized.xml'
+                lateralized = True
+            else:
+                label_file = 'HarvardOxford-Cortical.xml'
+        else:
+            label_file = 'HarvardOxford-Subcortical.xml'
     else:
-        label_file = 'HarvardOxford-Subcortical.xml'
+        label_file = "Juelich.xml"
         lateralized = False
+
     label_file = os.path.join(root, label_file)
 
-    atlas_file = os.path.join(root, 'HarvardOxford',
-                              'HarvardOxford-' + atlas_name + '.nii.gz')
+    atlas_file = os.path.join(root, atlas_source,
+                              '{}-{}.nii.gz'.format(atlas_source, atlas_name))
 
     atlas_img, label_file = _fetch_files(
         data_dir,
-        [(atlas_file, url, opts), (label_file, url, opts)],
+        [(atlas_file, urls[source_idx], opts),
+         (label_file, urls[source_idx], opts)],
```

Comment:
I realize now that this was here before but making `names` below a dict is confusing if we never use the keys